contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveDrillNativeParquetScan.java

```diff
@@ -213,11 +231,25 @@ protected boolean supportsFileImplicitColumns() {
     return hivePartitionHolder.get(locationProvider.getPath());
   }
 
+  @Override
+  protected ParquetMetadataProviderBuilder<?> tableMetadataProviderBuilder(MetadataProviderManager source) {
+    if (source.usesMetastore()) {
```

Comment:
When this case is possible?
When wee try to run analyze over Hive table using native parquet scan? Or when metastore is enabled and we try to query Hive table with native parquet scan? I really hope this is not the second case.